aten/src/ATen/core/jit_type.h

```diff
@@ -1869,7 +1869,13 @@ struct TORCH_API ClassType : public NamedType {
 
   std::string str() const override {
      return annotation_str();
-   }
+  }
+
+  std::string repr_str() const override {
+    std::stringstream ss;
+    ss << str() << "(" << compilation_unit().get() << ")";
```

Comment:
Nit:

Having a memory address trailing a type is puzzling too : )

How about this?
```
ss << str() << "(of Python compilation unit at: " << compilation_unit().get() << ")";
```

Reply:
Interesting idea. They could have names. In this case, it would be inferred from which imported module they come from. But I guess this is a slightly bigger change that has usability implications and warrants a discussion among team. What do you think?

Reply:
Yeah, makes sense, I can file and issue for it.